docs/core/additional-tools/wcf-web-service-reference-guide.md

```diff
@@ -73,9 +73,15 @@ When these processes complete, you can create an instance of the generated WCF c
 
 ## Next steps
 
+### See also
```

Comment:
I'd recommend ## See also and removing ## Next steps. Also I think Feedback & questions and Release notes can be H2s.

Reply:
Fixed